deb/Makefile

```diff
@@ -61,6 +61,19 @@ ubuntu-zesty: ## build ubuntu zesty deb packages
 		debbuild-$@/$(ARCH)
 	$(CHOWN) -R $(shell id -u):$(shell id -g) debbuild/$@
 
+.PHONY: ubuntu-artful
+ubuntu-artful: ## build ubuntu zesty deb packages
```

Comment:
In the comment: it's not zesty but artful :)

Reply:
Fixed! Thanks!